packages/aws.iac/src/resources/event-bridge/rule.ts

```diff
@@ -0,0 +1,149 @@
+import { resource } from "@notation/core";
+import { AwsSchema } from "src/utils/types";
+import * as sdk from "@aws-sdk/client-eventbridge";
+import z from "zod";
+import { eventBridgeClient } from "src/utils/aws-clients";
+import { LambdaFunctionInstance } from "../lambda";
+
+export type EventBridgeRuleSchema = AwsSchema<{
+  Key: sdk.DescribeRuleCommandInput;
+  // We omit the 'Rule' field since this is just an alias of the 'Name' field
+  CreateParams: sdk.PutRuleCommandInput &
+    Omit<sdk.PutTargetsCommandInput, "Rule">;
+  UpdateParams: sdk.PutRuleCommandInput &
+    Omit<sdk.PutTargetsCommandInput, "Rule">;
+  ReadResult: Omit<
+    sdk.DescribeRuleCommandOutput & sdk.ListTargetsByRuleCommandOutput,
+    "$metadata"
+  >;
+}>;
+
+export type EventBridgeRuleDependencies = {
+  lambda: LambdaFunctionInstance;
+};
+
+const eventBridgeRule = resource<EventBridgeRuleSchema>({
+  type: "aws/eventBridge/rule",
+});
+
+const targetSchema = z.object({
+  Id: z.string(),
+  Arn: z.string(),
+});
+
+const eventBridgeRuleSchema = eventBridgeRule.defineSchema({
+  Name: {
+    valueType: z.string(),
+    presence: "required",
+    primaryKey: true,
+    propertyType: "param",
+  },
+  EventBusName: {
+    valueType: z.string(),
+    presence: "optional",
```

Comment:
I think this might be achievable if we modify `SchemaItem` to have a 'default' field (and perhaps if we're being fancy it'd only be exposed if presence is 'optional'. Trying to use zod alone I'm running into this issue: https://github.com/colinhacks/zod/issues/2491

For what it's worth though, in this particular case I think 'optional' presence is fine as it's idiomatic to just omit the EventBusName parameter entirely and allow the AWS API to default for you.

I think this _will probably_ be an 'API layer concern' though once we introduce 'event bus rules' where the user _can_ pass in either their own event bus or use the default. There, I think we should think about the UX around this. However, in the `aws.iac` layer I think it's fine the way it is since that's how the API we're making a thin abstraction over behaves if that makes sense? :)